engine/src/physics/plugin.cpp

```diff
@@ -31,15 +48,40 @@ CUBOS_REFLECT_IMPL(Velocity)
         .build();
 }
 
+CUBOS_REFLECT_IMPL(AngularVelocity)
+{
+    return cubos::core::ecs::TypeBuilder<AngularVelocity>("cubos::engine::AngularVelocity")
+        .withField("vec", &AngularVelocity::vec)
+        .build();
+}
+
 CUBOS_REFLECT_IMPL(Force)
 {
-    return cubos::core::ecs::TypeBuilder<Force>("cubos::engine::Force").withField("mForce", &Force::mForce).build();
+    return cubos::core::ecs::TypeBuilder<Force>("cubos::engine::Force")
+        .withField("mForce", &Force::mForce)
+        .withField("mTorque", &Force::mTorque)
```

Comment:
Don't expose private fields through reflection - they're not meant to be modified directly. You can just reflect an empty type.